accepted/00102-git-integration.md

```diff
@@ -0,0 +1,63 @@
+- Feature Name: Git integration
+- Start Date: 2024-04-23
+
+# Summary
+[summary]: #summary
+
+This is a feature which will add a git integration to Uyuni without the need to use gitfs. It would be possible to sync formulas and playbooks and use them with an existent infrastructure.
+
+# Motivation
+[motivation]: #motivation
+
+- Why are we doing this?
+The git integration is not existent or it the depends from gitfs which is unstable
+
+- What use cases does it support?
+For all users which have a lot of self-written salt formulas and ansible-playbooks.
+
+- What is the expected outcome?
+Use salt formulas and ansible playbooks which are stored on git repositories
+Use them against clients managed by Uyuni
+
+
+# Detailed design
```

Comment:
What I think is missing: Such a solution should include that customers can add N git repositories. It is pretty unlikely that everything is in one large mono-repo.